src/main/java/org/moera/node/global/VirtualPageInterceptor.java

```diff
@@ -35,17 +35,10 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         VirtualPageHeader.put(response, virtualPage.value());
 
         if (requestContext.isBrowserExtension()) {
-            response.setContentType("text/html; charset=utf-8");
-            Writer out = new OutputStreamWriter(response.getOutputStream());
-            out.write("<!DOCTYPE html>\n"
-                    + "<html>\n"
-                    + "    <head>\n"
-                    + "        <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>\n"
-                    + "        <title>Moera</title>\n"
-                    + "    </head>\n"
-                    + "    <body></body>\n"
-                    + "</html>\n");
-            out.close();
+            response.setContentType("text/plain; charset=utf-8");
+            try (OutputStream outputStream = response.getOutputStream()) {
+                outputStream.write("Moera client is loading...".getBytes(StandardCharsets.UTF_8));
```

Comment:
You can use `OutputStreamWriter` with try-with-resources as well.

Reply:
I know, it just seams to me we don't need it.